src/Donations/PaymentMethods/PaymentFunctions.ts

```diff
@@ -537,13 +540,7 @@ export async function handleStripeSCAPayment({
             },
             billing_details: buildBillingDetails(contactDetails),
           },
-          return_url: `${
-            window.location.origin
-          }/?context=${donationID}&method=Sofort&tenant=${tenant}&country=${country}&locale=${
-            localStorage.getItem("language")
-              ? localStorage.getItem("language")
-              : "en"
-          }`,
+          return_url: `${window.location.origin}/?context=${donationID}&method=Sofort&tenant=${tenant}&country=${country}`,
```

Comment:
Same as above.

Reply:
Missed this out, will update.

Reply:
Fixed in 2cfc02e5dc3f0872b98c331fe5b99c849e684fe0